Bottom: bb02bd99c67f1a2740ffbf00c8d1c4bfffbceb37
Top:    13b86d0e7b9b6f9c44b40d9b12c409a8d77545d1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-06 15:59:22 +0000

Refresh of polygon1-c-fix-collect-routine

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 95ce14f..f721685 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1664,6 +1664,10 @@ Collect (char poly, jmp_buf * e, PLINE * a, POLYAREA ** contours, PLINE ** holes
 
       if (j_rule (poly, cur, &dir) && cur->Flags.mark == 0)
 	Collect1 (e, cur, dir, contours, holes, j_rule);
+//    The lines below are no-longer required.. preserving for now to record this commit.
+//      other = cur;
+//      if (other->cvc_prev)
+//	Collect1 (e, other, dir, contours, holes, j_rule);
     }
   while ((cur = cur->next) != &a->head);
 }				/* Collect */
